tasks.py

```diff
@@ -74,7 +76,7 @@ def lint(ctx: Context, check: bool = False, unsafe_fixes: bool = False) -> None:
 
 
 @invoke.task(
-    aliases=["build"],
+    aliases=("build",),
```

Comment:
`invoke` is looking for a tuple here instead of a list. Noticed in the new code, fixed all the existing usages in this file.